src/index.js

```diff
@@ -0,0 +1,158 @@
+      eachcheck[lis.index].classList.add('checkbox');
+      eachcheck[lis.index].setAttribute('id', lis.index);
+
+      // create paragraph
+      eachParag[lis.index] = document.createElement('p');
+      /// ///////////////////////////////
+      eachParag[lis.index].textContent = lis.description;
+      eachParag[lis.index].setAttribute('id', lis.index);
+      eachParag[lis.index].classList.add('eachParag');
+
+      // create button
+      eachbtn[lis.index] = document.createElement('button');
+      eachbtn[lis.index].setAttribute('id', lis.index);
+      eachbtn[lis.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
+
+      // Div
+      checkNparagDiv[lis.index] = document.createElement('div');
+      checkNparagDiv[lis.index].append(eachcheck[lis.index], eachParag[lis.index]);
+      eachParag[lis.index].classList.add('checkNparagDiv');
+
+      // appending check, paragraph and button
+      eachList[lis.index].append(eachcheck[lis.index], eachParag[lis.index], eachbtn[lis.index]);
+
+      // appending each list
+      todos[0].append(eachList[lis.index]);
+
+      // On editing paragraph
+      eachParag[lis.index].addEventListener('click', (e) => {
+        e.target.nextSibling.innerHTML = '<i class="fa-solid fa-trash"></i>';
+        e.target.nextSibling.style.cursor = 'pointer';
+        eachList[lis.index].style.backgroundColor = '#a6bdbe';
+        e.target.nextSibling.addEventListener('click', () => {
+          this.delete(eachList[lis.index]);
+        });
+      });
+
+      eachList[lis.index].addEventListener('mouseleave', (eve) => {
+        eachbtn[lis.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
+        this.update(eve.target.id, eve.target.innerText);
+      });
+
+      // Listen for checking elements
+      eachcheck[lis.index].addEventListener('change', (e) => {
+        const status = new Status();
+        if (e.target.checked === true) {
+          status.checked(this.doings[lis.index]);
+        } else {
+          status.unchecked(this.doings[lis.index]);
+        }
+
+        this.update(e.target.nextSibling.id, e.target.nextSibling.innerText);
+      });
+
+      // conditions after checking
+      if (this.doings[lis.index].completed === true) {
+        eachcheck[lis.index].setAttribute('checked', 'checked');
+        eachList[lis.index].classList.add('checked');
+      } else if (this.doings[lis.index].completed === false) {
+        eachcheck[lis.index].removeAttribute('checked');
+        eachList[lis.index].classList.remove('checked');
+      }
+    });
+  }
+
+  add(description, completed, index) {
+    const allTask = new Alltask(description, completed, index);
+    this.doings.push(allTask);
+  }
+
+  delete(num) {
+    const key = num;
+    if (this.doings.length === 1) {
+      this.doings = [];
+    } else {
+      this.doings.splice(key, 1);
+    }
+    this.doings.forEach((lis, index) => {
+      lis.index = index;
+    });
```

Comment:
- [ ] Make sure that you implement very well your method to remove a task from your ToDo List so that we can process the CRUD operation. For now, I can't see an icon for deleting a task unless I want to edit a task. Kindly, fix this issue.

Reply:
DONE